### ISSUE TITLE
The page is readable and functional when text is enlarged up to 200%.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#147]
Text does not adapt when users adjust the text size up to 200% using the browser settings in Chrome at the 1280x1024 viewport. We recommend switching fonts, related styles, and media queries to em or rem approaches to allow Chrome to adjust layouts and font sizes as users increase font styles.

When the viewport is zoomed to 200% on a 1280x1024 display in Firefox, the image overlaps the "A Tradition...Possibility" heading text on the page, making it difficult to reach the content. It should be ensured that this content is accessible to all users who use Zoom.A reference image is available at the end of this page section

**Comment by justpie** [comment]
@negilles  - I'm unable to reproduce any issues at 200%. Does this happen for you?
